ow_lander/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ add_action_files(
   FILES GuardedMove.action
   FILES AntennaPanTilt.action
   FILES Discard.action
+  FILES Stop.action

```

Comment:
I think a more specific name than "Stop" would be good, but let's hold off on renaming until we address naming across the whole OW/OWLAT interface.